static/app/components/searchSyntax/parser.tsx

```diff
@@ -415,6 +416,14 @@ export class TokenConverter {
     };
   };
 
+  tokenParen = (value: '(' | ')') => ({
+    ...this.defaultTokenFields,
+    value,
+    left: value === '(',
+    right: value === ')',
+    type: Token.PAREN as const,
```

Comment:
I would just use Token.LPAREN | Token.RPAREN
```suggestion
    type: Token.LPAREN | Token.RPAREN
```
The left/right fields will end up in code like Token.type === Token.PAREN && token.left which isnt terrible, but makes it a bit less ergonomic if you were to use the Token.Paren in something like a switch statement.

Reply:
Ah yeah, I do like that better. Changed